Intermittent failing unit test in content-server: views/sign_up->submit->COPPA is valid->signup succeeds

The [check for the `firefox-accounts-journey` array](https://github.com/mozilla/fxa/blob/a71351f020398868fd4fe0d497be49fdd0e02fa2/packages/fxa-content-server/app/tests/spec/views/sign_up.js#L878) fails about 1 time in 2.

>    views/sign_up
        submit
            COPPA is valid
                signup succeeds
                    calls view.signUp correctly, does not display any errors ‣

                    AssertionError: expected undefined to be an arrayAssertionError@http://127.0.0.1:3030/bundle/appDependencies.bundle.js:8429:13
                    ../../node_modules/chai/lib/chai/assertion.js/module.exports/Assertion.prototype.assert@http://127.0.0.1:3030/bundle/appDependencies.bundle.js:13089:13
                    an@http://127.0.0.1:3030/bundle/appDependencies.bundle.js:13535:10
                    chainableMethodWrapper@http://127.0.0.1:3030/bundle/appDependencies.bundle.js:20568:49
                    ../../node_modules/chai/lib/chai/core/assertions.js/module.exports/<@http://127.0.0.1:3030/bundle/appDependencies.bundle.js:16303:51
                    methodWrapper@http://127.0.0.1:3030/bundle/appDependencies.bundle.js:20746:25
                    ../../node_modules/chai/lib/chai/interface/assert.js/module.exports/assert.sameMembers@http://127.0.0.1:3030/bundle/appDependencies.bundle.js:19193:21
                    ../tests/spec/views/sign_up.js/</</</</<@http://127.0.0.1:3030/bundle/test.bundle.js:50378:55
                    callFn@http://127.0.0.1:3030/bundle/appDependencies.bundle.js:48904:21
                    next/<@http://127.0.0.1:3030/bundle/appDependencies.bundle.js:49509:12
                    next@http://127.0.0.1:3030/bundle/appDependencies.bundle.js:49305:14
                    next/<@http://127.0.0.1:3030/bundle/appDependencies.bundle.js:49315:11
                    next@http://127.0.0.1:3030/bundle/appDependencies.bundle.js:49239:14
                    next/<@http://127.0.0.1:3030/bundle/appDependencies.bundle.js:49278:11
                    done@http://127.0.0.1:3030/bundle/appDependencies.bundle.js:48851:7
                    callFn/<@http://127.0.0.1:3030/bundle/appDependencies.bundle.js:48909:15
                    next@http://127.0.0.1:3030/bundle/appDependencies.bundle.js:49308:10
                    next/<@http://127.0.0.1:3030/bundle/appDependencies.bundle.js:49315:11
                    next@http://127.0.0.1:3030/bundle/appDependencies.bundle.js:49239:14
                    next/<@http://127.0.0.1:3030/bundle/appDependencies.bundle.js:49278:11
                    done@http://127.0.0.1:3030/bundle/appDependencies.bundle.js:48851:7
                    callFn/<@http://127.0.0.1:3030/bundle/appDependencies.bundle.js:48909:15
                    next@http://127.0.0.1:3030/bundle/appDependencies.bundle.js:49308:10
                    next/</<@http://127.0.0.1:3030/bundle/appDependencies.bundle.js:49541:14
                    done@http://127.0.0.1:3030/bundle/appDependencies.bundle.js:48851:7
                    callFn@http://127.0.0.1:3030/bundle/appDependencies.bundle.js:48922:11
                    next/<@http://127.0.0.1:3030/bundle/appDependencies.bundle.js:49509:12
                    next@http://127.0.0.1:3030/bundle/appDependencies.bundle.js:49305:14
                    next/<@http://127.0.0.1:3030/bundle/appDependencies.bundle.js:49315:11
                    next@http://127.0.0.1:3030/bundle/appDependencies.bundle.js:49239:14
                    next/<@http://127.0.0.1:3030/bundle/appDependencies.bundle.js:49278:11
                    done@http://127.0.0.1:3030/bundle/appDependencies.bundle.js:48851:7
                    callFn/<@http://127.0.0.1:3030/bundle/appDependencies.bundle.js:48909:15
                    next@http://127.0.0.1:3030/bundle/appDependencies.bundle.js:49308:10
                    next/<@http://127.0.0.1:3030/bundle/appDependencies.bundle.js:49315:11
                    next@http://127.0.0.1:3030/bundle/appDependencies.bundle.js:49239:14
                    next/<@http://127.0.0.1:3030/bundle/appDependencies.bundle.js:49278:11
                    done@http://127.0.0.1:3030/bundle/appDependencies.bundle.js:48851:7
                    callFn/<@http://127.0.0.1:3030/bundle/appDependencies.bundle.js:48909:15
                    next@http://127.0.0.1:3030/bundle/appDependencies.bundle.js:49308:10
                    next/</<@http://127.0.0.1:3030/bundle/appDependencies.bundle.js:49541:14
                    done@http://127.0.0.1:3030/bundle/appDependencies.bundle.js:48851:7
                    callFn@http://127.0.0.1:3030/bundle/appDependencies.bundle.js:48922:11
                    next/<@http://127.0.0.1:3030/bundle/appDependencies.bundle.js:49509:12
                    next@http://127.0.0.1:3030/bundle/appDependencies.bundle.js:49305:14
                    next/<@http://127.0.0.1:3030/bundle/appDependencies.bundle.js:49315:11
                    next@http://127.0.0.1:3030/bundle/appDependencies.bundle.js:49239:14
                    next/<@http://127.0.0.1:3030/bundle/appDependencies.bundle.js:49278:11
                    done@http://127.0.0.1:3030/bundle/appDependencies.bundle.js:48851:7
                    callFn/<@http://127.0.0.1:3030/bundle/appDependencies.bundle.js:48909:15
                    next@http://127.0.0.1:3030/bundle/appDependencies.bundle.js:49308:10
                    next/<@http://127.0.0.1:3030/bundle/appDependencies.bundle.js:49315:11
                    next@http://127.0.0.1:3030/bundle/appDependencies.bundle.js:49239:14
                    next/<@http://127.0.0.1:3030/bundle/appDependencies.bundle.js:49278:11
                    done@http://127.0.0.1:3030/bundle/appDependencies.bundle.js:48851:7
                    callFn/<@http://127.0.0.1:3030/bundle/appDependencies.bundle.js:48909:15
                    next@http://127.0.0.1:3030/bundle/appDependencies.bundle.js:49308:10

1

--- a/packages/fxa-content-server/app/tests/spec/views/sign_up.js
+++ b/packages/fxa-content-server/app/tests/spec/views/sign_up.js
@@ -843,6 +843,7 @@ describe('views/sign_up', function() {
         beforeEach(function() {
           sinon.stub(view, 'signUp').callsFake(() => Promise.resolve());
           sinon.stub(view, '_hasOptedIntoNewsletter').callsFake(() => true);
+          sinon.stub(view, 'isAnyNewsletterVisible').callsFake(() => true);
 
           return view.submit();
         });

